docs/mdbook-admonish.css

```diff
@@ -0,0 +1,351 @@
+@charset "UTF-8";
```

Comment:
Did you get a template from somewhere? Should there be a comment explaining where one could start if they wanted to make changes here? (Although I guess this file would be rarely if ever changed)